t/40_sxc.t

```diff
@@ -30,13 +30,18 @@ my $content;
     #like ($@, qr/too short/);
     }
 
+my $sxc_fh;
+{   open $sxc_fh, '<', "files/test.sxc" or die "files/test.sxc: $!\n";
+    }
 foreach my $base ( [ "files/test.sxc",		"Read/Parse sxc file" ],
 		   [ "files/content.xml",	"Read/Parse xml file" ],
 		   [ $content,			"Parse xml data" ],
+		   [ $sxc_fh,			"Parse FH" ],
 		   ) {
     my ($txt, $msg) = @$base;
     my $sxc;
-    ok ($sxc = ReadData ($txt), $msg);
+    my @options = ref $txt ? ( parser => 'ods'): ();
```

Comment:
I prefer double quotes for strings: `my @options = ref $txt ? (parser => "ods") : ();` or `my @options = ref $txt ? (parser => "ods") : ();`